src/schema.sql

```diff
@@ -5,28 +5,44 @@ CREATE EXTENSION IF NOT EXISTS pgcrypto;
 CREATE EXTENSION IF NOT EXISTS hstore;
 
 DROP TABLE IF EXISTS webhooks;
+DROP TABLE IF EXISTS meta;
+DROP TABLE IF EXISTS users_tokens;
+DROP TABLE IF EXISTS bounds;
+DROP TABLE IF EXISTS tiles;
+DROP TABLE IF EXISTS geo;
+DROP TABLE IF EXISTS geo_history;
+DROP TABLE IF EXISTS styles;
+DROP TABLE IF EXISTS deltas;
+DROP TABLE IF EXISTS users;
```

Comment:
This  re-org makes the PR hard to follow. Can we unroll this, or at least get the changes into a single commit that can be viewed on it's own?

Reply:
@miccolis Can't unroll this now as due to the addition of proper references, this needs to be at the top and in the correct order or schema application will fail. I'll defer to @lizziegooding for rolling into a single commit

Reply:
Removed in https://github.com/mapbox/Hecate/pull/206/commits/7493e0efafaeec0af7b70cbab56df3f1b16525d5 -- while these are changes we should make, i'm removing them from the current PR to reduce scope for what will already be a large change.